### ISSUE TITLE
400 error when changing a username

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#2186]
In `/management/facility#/users`...

1. create a new coach user with username `username11`:

    ![image](https://user-images.githubusercontent.com/2367265/30396962-29da09fa-9880-11e7-8aff-e41327a3f997.png)

2. edit the user:

    ![image](https://user-images.githubusercontent.com/2367265/30396987-3edcdb7a-9880-11e7-82f8-534cc853bfa7.png)

3. change username to `username2`:

    ![image](https://user-images.githubusercontent.com/2367265/30397035-5c9b0146-9880-11e7-9c00-ea9a7d4a678a.png)

operation fails with `400`:

```
{"non_field_errors":["The fields user, collection, kind must make a unique set."]}
```

**Comment by christianmemije** [comment]
Can you replicate in #2154 ?

**Comment by indirectlylit** [comment]
nope, appears to be addressed there!

**Comment by christianmemije** [comment]
Fixed in #2154
